jenkins-agents/jenkins-agent-python-and-swig/Jenkinsfile.test

```diff
@@ -0,0 +1,66 @@
+pipeline {
+    agent {
+      label 'jenkins-agent-python-and-swig'
+    }
+
+    stages {
+        stage ('Run Test') {
+            steps {
+              sh """
+                  python -V
+                  gcc --version
+                  swig -version
+              """
+              // sh """
```

Comment:
Can you please remove all the commented out code, if they are not needed ?

Reply:
Sure!